### DISCUSSION TITLE
Announce: bash-env plugin updated

### DISCUSSION THREAD

**Discussion by tesujimath** [nushell/nushell#11675] [General]
Thanks to @Abdillah there's a version bump to my [bash-env plugin](https://github.com/tesujimath/nu_plugin_bash_env), which now supports environment import via stdin.

So e.g.
```
> ssh-agent | bash-env | load-env
```

Feedback welcome.

**Comment by fdncred** [comment]
Is there some secret way to clone the repo? Neither `gix` nor `git` like it for some reason (on Windows). I bet it's a Windows issue file/directory naming problem.
```
❯ gix clone https://github.com/tesujimath/nu_plugin_bash_env.git
 15:46:16 indexing done 94.0 objects in 0.01s (16.3k objects/s)
 15:46:16 decompressing done 30.5KB in 0.01s (4.7MB/s)
 15:46:16     Resolving done 94.0 objects in 0.12s (782.0 objects/s)
 15:46:16      Decoding done 146.5KB in 0.12s (1.2MB/s)
 15:46:16 writing index file done 3.7KB in 0.00s (7.2MB/s)
 15:46:16  create index file done 94.0 objects in 0.13s (724.0 objects/s)
 15:46:16          read pack done 17.4KB in 0.19s (90.6KB/s)
Error: IO error while writing blob or reading file metadata or changing filetype

Caused by:
    The filename, directory name, or volume label syntax is incorrect. (os error 123)
❯ git clone https://github.com/tesujimath/nu_plugin_bash_env.git
Cloning into 'nu_plugin_bash_env'...
remote: Enumerating objects: 94, done.
remote: Counting objects: 100% (94/94), done.
remote: Compressing objects: 100% (51/51), done.
Receiving objects: 100% (94/94), 16.99 KiB | 241.00 KiB/s, done.

Resolving deltas: 100% (50/50), done.
error: invalid path 'examples/Ming's "menu" of (merciless) monstrosities.env'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
```

**Reply by fdncred** [reply]
I got it checked out in linux. I'm specifically interested in your rust api-generator project. Something generic like that would be helpful to anyone writing a plugin in other languages, especially languages like python, bash, and other scripting type languages.

**Reply by tesujimath** [reply]
Windows clearly doesn't like my example pathological filenames in examples. Which I will accept as a commendation!

Sorry about that!

**Reply by tesujimath** [reply]
@fdncred I think it was the double quotes.  Now removed in the latest version, so hopefully you can clone the repo on Windows now. :crossed_fingers:

**Reply by fdncred** [reply]
I can clone it now. Thanks!
